### ISSUE TITLE
Hive - Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.StatsTask

### ISSUE THREAD

**Issue by hamdiazz** [TOSIT-IO/hive#9]
I have an error when i insert a line in a table on Hive 3.1.3, also, i found this bug below:

https://issues.apache.org/jira/browse/HIVE-19316

```hiveql
ERROR [4f8550ab-6756-4886-9401-93fde03bba46 main] ql.Driver: FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.StatsTask 
``` 
After applying the modification in 16 files like mentioned in the PR https://github.com/apache/hive/pull/378/files,  the error disappeared.

**Comment by rpignolet** [comment]
The commit to backport is https://github.com/apache/hive/commit/bfa69e1077a61668b69f54dcc54609f773f89c93

**Comment by hamdiazz** [comment]
Hi,
Here is below the patch :
https://github.com/apache/hive/commit/bfa69e1077a61668b69f54dcc54609f773f89c93.patch

but the path "standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/columnstats" has been replaced by "standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats".

we find below errors when i launch a git apply of the patch :

```
git apply patches/bfa69e1077a61668b69f54dcc54609f773f89c93.patch
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/DateColumnStatsAggregator.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/DecimalColumnStatsAggregator.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/DoubleColumnStatsAggregator.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/LongColumnStatsAggregator.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/StringColumnStatsAggregator.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/cache/DateColumnStatsDataInspector.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/cache/DecimalColumnStatsDataInspector.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/cache/DoubleColumnStatsDataInspector.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/cache/LongColumnStatsDataInspector.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/cache/StringColumnStatsDataInspector.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/DateColumnStatsMerger.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/DecimalColumnStatsMerger.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/DoubleColumnStatsMerger.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/LongColumnStatsMerger.java: Aucun fichier ou dossier de ce type
error: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/StringColumnSt
```

**Comment by kpgtek** [comment]
After making few tests on transactional tables (ORC format), I would like to add some information for this issue.
I have followed this article [update-on-hive-1-2-1](https://stackoverflow.com/questions/40677208/update-on-hive-1-2-1) to enable HIVE compaction in order to use ORC tables with transactional state to delete and update rows.
And in my HIVE logs, it seems that the error mentionned in this issue would disappear when inserting rows in the table.

**Comment by PACordonnier** [comment]
Fix is included in current 3.1.3-2.0-SNAPSHOT
